ccd-definition/CaseField.json

```diff
@@ -354,6 +362,15 @@
     "HintText": "For example, joe.bloggs@la.gov.uk",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "gateKeeperEmails",
+    "Label": "Gatekeeper",
+    "FieldType": "Collection",
+    "FieldTypeParameter": "GateKeeperEmail",
```

Comment:
Using Complex type as using

`  {
    "LiveFrom": "01/01/2017",
    "CaseTypeID": "CARE_SUPERVISION_EPO",
    "ID": "gateKeeperEmails",
    "Label": "Gatekeeper",
    "FieldType": "Collection",
    "FieldTypeParameter": "Email",
    "SecurityClassification": "Public"
  },`

Uses Gatekeeper 1, 2, 3 etc as text input heading rather than email. Also fails to perform client side validation.